### PR TITLE
Rename packages

### PR DESCRIPTION
**Pull request by boxhock**

### CODE REVIEW

File: blockchain.com/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@chainlink/blockchain.com",
+  "name": "@chainlink/blockchain.com-adapter",
```
Comment (krebernisak) [inline blockchain.com/package.json:2]: We should probably depreciate already published packages, maybe [using this tool to make it easy](https://stackoverflow.com/a/56197360). We can maybe also reset the version to `0.0.1`, what do you think?

**Review by krebernisak — APPROVED** [review]

File: blockchain.com/package.json (hunk 1 above)
Comment (boxhock) [reply blockchain.com/package.json:2]: I'm not sure if it's necessary to do it, as only a handful of adapters have been published so far, a few of them were published with "-adapter" already appended, and afaik, only we are using these packages. I don't think any of the versions have ever been upgraded either.

If you're aware of any packages/adapters that have been published with a version > 0.0.1, we can probably update them.

Comment (krebernisak) [reply blockchain.com/package.json:2]: Yeah, I think that PoR adapters are on `0.0.3` as we messed up the TS builds a few times and published the package without types.

File: covid-tracker/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@chainlink/covid-tracker",
+  "name": "@chainlink/covid-tracker-adapter",
   "version": "0.1.0",
```
Comment (krebernisak) [inline covid-tracker/package.json:3]:
```suggestion
  "version": "0.0.1",
```

File: lition/package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@chainlink/lition",
+  "name": "@chainlink/lition-adapter",
   "version": "0.1.0",
```
Comment (krebernisak) [inline lition/package.json:3]:
```suggestion
  "version": "0.0.1",
```

File: messari/package.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@chainlink/messari",
+  "name": "@chainlink/messari-adapter",
   "version": "0.1.0",
```
Comment (krebernisak) [inline messari/package.json:3]:
```suggestion
  "version": "0.0.1",
```

File: coinlore/package.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@chainlink/coinlore",
+  "name": "@chainlink/coinlore-adapter",
   "version": "0.1.0",
```
Comment (krebernisak) [inline coinlore/package.json:3]:
```suggestion
  "version": "0.0.1",
```

File: blockchain.com/package.json [hunk 6]
```diff
@@ -1,5 +1,5 @@
...
   "version": "0.0.3",
```
Comment (krebernisak) [inline blockchain.com/package.json:3]:
```suggestion
  "version": "0.0.1",
```

File: blockcypher/package.json [hunk 7]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@chainlink/blockcypher",
+  "name": "@chainlink/blockcypher-adapter",
   "version": "0.0.3",
```
Comment (krebernisak) [inline blockcypher/package.json:3]:
```suggestion
  "version": "0.0.1",
```

File: reduce/package.json [hunk 8]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@chainlink/reduce",
+  "name": "@chainlink/reduce-adapter",
   "version": "0.0.3",
```
Comment (krebernisak) [inline reduce/package.json:3]:
```suggestion
  "version": "0.0.1",
```

File: renvm-address-set/package.json [hunk 9]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@chainlink/renvm-address-set",
+  "name": "@chainlink/renvm-address-set-adapter",
   "version": "0.0.3",
```
Comment (krebernisak) [inline renvm-address-set/package.json:3]:
```suggestion
  "version": "0.0.1",
```

File: wbtc-address-set/package.json [hunk 10]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@chainlink/wbtc-address-set",
+  "name": "@chainlink/wbtc-address-set-adapter",
   "version": "0.0.3",
```
Comment (krebernisak) [inline wbtc-address-set/package.json:3]:
```suggestion
  "version": "0.0.1",
```

**Review by krebernisak — CHANGES_REQUESTED** [review]

File: blockchain.com/package.json (hunk 1 above)
Comment (boxhock) [reply blockchain.com/package.json:2]: Thanks for adding the suggestions, really helpful!

**Review by krebernisak — APPROVED** [review]

**Pull request merged**